cloudinit/config/cc_growpart.py

```diff
@@ -210,11 +210,35 @@ def get_size(filename):
 
 def device_part_info(devpath):
     # convert an entry in /dev/ to parent disk and partition number
+    myenv = os.environ.copy()
+    myenv['LANG'] = 'C'
 
     # input of /dev/vdb or /dev/disk/by-label/foo
     # rpath is hopefully a real-ish path in /dev (vda, sdb..)
     rpath = os.path.realpath(devpath)
 
+    is_lvm = True if re.search('.*dm.*', rpath) else False
+    if is_lvm:
+        try:
+            (out, _err) = subp.subp(["lvm", "lvs", devpath, "-o", "vgname",
+                                     "--noheadings"], env=myenv)
```

Comment:
setting LANG is good if you're processing output.  thank you.
you can do it without the full copy though.

just use `update_env`

```python
myenv = {'LANG': 'C'}
subp.subp(["lvm", "lvs", ...], update_env=env)
```

and also `--output=vgname`

Reply:
```
[root@huzhaolvmfordev01220919-vm1 ~]# lvm vgs rootvg --output vgname  --noheadings
vgs: unrecognized option '--output'
  Error during parsing of command line.
```
Currently I'm testing on a `RHEL-8.3` with LVM `2.03.09(2)-RHEL8`:
```
[root@huzhaolvmfordev01220919-vm1 ~]# lvm version
  LVM version:     2.03.09(2)-RHEL8 (2020-05-28)
  Library version: 1.02.171-RHEL8 (2020-05-28)
  Driver version:  4.42.0
  Configuration:   ./configure --build=x86_64-redhat-linux-gnu --host=x86_64-redhat-linux-gnu --program-prefix= --disable-dependency-tracking --prefix=/usr --exec-prefix=/usr --bindir=/usr/bin --sbindir=/usr/sbin --sysconfdir=/etc --datadir=/usr/share --includedir=/usr/include --libdir=/usr/lib64 --libexecdir=/usr/libexec --localstatedir=/var --sharedstatedir=/var/lib --mandir=/usr/share/man --infodir=/usr/share/info --with-default-dm-run-dir=/run --with-default-run-dir=/run/lvm --with-default-pid-dir=/run --with-default-locking-dir=/run/lock/lvm --with-usrlibdir=/usr/lib64 --enable-fsadm --enable-write_install --with-user= --with-group= --with-device-uid=0 --with-device-gid=6 --with-device-mode=0660 --enable-pkgconfig --enable-cmdlib --enable-dmeventd --enable-blkid_wiping --with-cluster=internal --enable-cmirrord --with-udevdir=/usr/lib/udev/rules.d --enable-udev_sync --with-thin=internal --with-cache=internal --enable-lvmpolld --enable-lvmlockd-dlm --enable-lvmlockd-dlmcontrol --enable-lvmlockd-sanlock --enable-dbus-service --enable-notify-dbus --enable-dmfilemapd --with-writecache=internal --with-vdo=internal --with-vdo-format=/usr/bin/vdoformat --with-integrity=internal --disable-silent-rules
```